### ISSUE TITLE
Conflict installing with django==3.1.5

### ISSUE THREAD

**Issue by shuttle1987** [celiao/django-rest-authemail#31]
At the moment I'm attempting to install this package using the following `requirements.txt` file:

```
Django==3.1.5
djangorestframework==3.12.2
python-decouple==3.4
drf-spectacular==0.13.1
django-rest-authemail==2.0.6
```

This is failing with the following:

```
Step 7/8 : RUN pip install -r requirements.txt
 ---> Running in 769a8a17f147
Collecting Django==3.1.5
  Downloading Django-3.1.5-py3-none-any.whl (7.8 MB)
Collecting djangorestframework==3.12.2
  Downloading djangorestframework-3.12.2-py3-none-any.whl (957 kB)
Collecting python-decouple==3.4
  Downloading python_decouple-3.4-py3-none-any.whl (9.5 kB)
Collecting drf-spectacular==0.13.1
  Downloading drf_spectacular-0.13.1-py2.py3-none-any.whl (59 kB)
Collecting django-rest-authemail==2.0.6
  Downloading django_rest_authemail-2.0.6-py3-none-any.whl (41 kB)
Collecting sqlparse>=0.2.2
  Downloading sqlparse-0.4.1-py3-none-any.whl (42 kB)
Collecting pytz
  Downloading pytz-2020.5-py2.py3-none-any.whl (510 kB)
Collecting asgiref<4,>=3.2.10
  Downloading asgiref-3.3.1-py3-none-any.whl (19 kB)
INFO: pip is looking at multiple versions of <Python from Requires-Python> to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of django to determine which version is compatible with other requirements. This could take a while.
ERROR: Cannot install -r requirements.txt (line 5) and Django==3.1.5 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested Django==3.1.5
    django-rest-authemail 2.0.6 depends on Django<=3.1 and >=2.2.8

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/user_guide/#fixing-conflicting-dependencies
ERROR: Service 'backend' failed to build : The command '/bin/sh -c pip install -r requirements.txt' returned a non-zero code: 1
```

I notice that in the `setup.py` file for this project:

https://github.com/celiao/django-rest-authemail/blob/5eb1a2199e6159a9f3c144162e876b1f9a636917/setup.py#L30-L34

The version requirement here is pinned against `Django<=3.1`. If Django 3.1 is supported, as I suspect is the case since #27 has been closed would this issue be fixed by changing this requirement in setup.py to `Django<3.2`?

**Comment by shuttle1987** [comment]
Now I'm aware that my particular `requirements.txt` file will fail to install since this package has an exact dependency on 'djangorestframework==3.11.0', but the issue with Django versions I think is independent of this.

**Comment by arielaco** [comment]
Receiving similar error.

`ERROR: django-rest-authemail 2.0.6 has requirement djangorestframework==3.11.0, but you'll have djangorestframework 3.12.4 which is incompatible.`

**Comment by Avin-Techv** [comment]
Receiving similar error.

`ERROR: Cannot install -r requirements.txt (line 16), django-rest-authemail==2.0.6 and djangorestframework==3.12.4 because these package versions have conflicting dependencies.`

**Comment by Patrykx85** [comment]
I fix it in pull request 42

**Comment by celiao** [comment]
This issue should be addressed by PR #42, which I just pulled
